app/src/main/java/com/example/android/binge/Series.java

```diff
@@ -0,0 +1,172 @@
+package com.example.android.binge;
```

Comment:
Some of these files may not be needed, such as Series.java. This overlaps with what I had already committed to handling. This can be merged and we can remove any unnecessary files once master is in a stable state (after I submit a PR after yours is approved and merged).